From 358a5f74869257b537936c0bd8333bbfa0406d6f Mon Sep 17 00:00:00 2001
From: Abinaya <abinaya.dhandapani@amd.com>
Date: Thu, 10 Feb 2022 12:14:47 -0600
Subject: [PATCH 1/1] ARM:dts:aspeed: Device tree changes for APML_ALERT_L

Device tree changes added for APML_ALERT_L and removed sbrmi driver from
device tree as it is not being used.

Signed-off-by: Abinaya <abinaya.dhandapani@amd.com>
---
 arch/arm/boot/dts/aspeed-bmc-amd-onyx.dts     | 11 +--------
 arch/arm/boot/dts/aspeed-bmc-amd-quartz.dts   | 22 ++---------------
 arch/arm/boot/dts/aspeed-bmc-amd-ruby.dts     | 11 +--------
 arch/arm/boot/dts/aspeed-bmc-amd-titanite.dts | 24 ++-----------------
 4 files changed, 6 insertions(+), 62 deletions(-)

diff --git a/arch/arm/boot/dts/aspeed-bmc-amd-onyx.dts b/arch/arm/boot/dts/aspeed-bmc-amd-onyx.dts
index 97ed24d8fa6d..6af4289c017b 100644
--- a/arch/arm/boot/dts/aspeed-bmc-amd-onyx.dts
+++ b/arch/arm/boot/dts/aspeed-bmc-amd-onyx.dts
@@ -741,10 +741,6 @@ sbtsi_p0_0: sbtsi@4c,22400000000 {
         reg = <0x4c 0x224 0x00000000>;
         assigned-address = <0x4c>;
     };
-    sbrmi_p0_1: sbrmi@3c,22400000001 {
-        reg = <0x3c 0x224 0x00000001>;
-        assigned-address = <0x3c>;
-    };
 };

 #endif  // OPENBMC_I3C_ENABLE
@@ -758,11 +754,6 @@ sbtsi@4c {
 		compatible = "amd,sbtsi";
 		reg = <0x4c>;
 	};
-	// sbrmi
-	sbrmi@3c {
-		compatible = "amd,sbrmi";
-		reg = <0x3c>;
-	};
 };

 &espi_ctrl {
@@ -788,7 +779,7 @@ &gpio0 {
 	/*F0-F7*/	"","","","","","","","",
 	/*G0-G7*/	"","","","","","","","",
 	/*H0-H7*/	"","ASSERT_WARM_RST_BTN_L","ASSERT_SOC_RST_BTN_L","","","","","",
-	/*I0-I7*/	"","","","","","","","",
+	/*I0-I7*/	"","","","","","","","P0_I3C_APML_ALERT_L",
 	/*J0-J7*/	"","","","","","","","",
 	/*K0-K7*/	"","","","","","","","",
 	/*L0-L7*/	"","","","","","","","",
diff --git a/arch/arm/boot/dts/aspeed-bmc-amd-quartz.dts b/arch/arm/boot/dts/aspeed-bmc-amd-quartz.dts
index 5431de818ac6..e0a4d1178c38 100644
--- a/arch/arm/boot/dts/aspeed-bmc-amd-quartz.dts
+++ b/arch/arm/boot/dts/aspeed-bmc-amd-quartz.dts
@@ -1030,10 +1030,6 @@ sbtsi_p0_0: sbtsi@4c,22400000000 {
         reg = <0x4c 0x224 0x00000000>;
         assigned-address = <0x4c>;
     };
-    sbrmi_p0_1: sbrmi@3c,22400000001 {
-        reg = <0x3c 0x224 0x00000001>;
-        assigned-address = <0x3c>;
-    };
 };

 &i3c5 {
@@ -1050,10 +1046,6 @@ sbtsi_p1_0: sbtsi@4c,22400000000 {
         reg = <0x4c 0x224 0x00000000>;
         assigned-address = <0x4c>;
     };
-    sbrmi_p1_1: sbrmi@3c,22400000001 {
-        reg = <0x3c 0x224 0x00000001>;
-        assigned-address = <0x3c>;
-    };
  };

 #endif  // OPENBMC_I3C_ENABLE
@@ -1067,11 +1059,6 @@ sbtsi@4c {
 		compatible = "amd,sbtsi";
 		reg = <0x4c>;
 	};
-	// sbrmi
-	sbrmi@3c {
-		compatible = "amd,sbrmi";
-		reg = <0x3c>;
-	};
 };

 &i2c3 {
@@ -1083,11 +1070,6 @@ sbtsi@48 {
 		compatible = "amd,sbtsi";
 		reg = <0x48>;
 	};
-	// sbrmi
-	sbrmi@38 {
-		compatible = "amd,sbrmi";
-		reg = <0x38>;
-	};
 };

 &espi_ctrl {
@@ -1113,7 +1095,7 @@ &gpio0 {
 	/*F0-F7*/	"","","","","","","","",
 	/*G0-G7*/	"","","","","","","","",
 	/*H0-H7*/	"","ASSERT_WARM_RST_BTN_L","ASSERT_SOC_RST_BTN_L","","","","","",
-	/*I0-I7*/	"","","","","","","","",
+	/*I0-I7*/	"","","","","","","","P0_I3C_APML_ALERT_L",
 	/*J0-J7*/	"","","","","","","","",
 	/*K0-K7*/	"","","","","","","","",
 	/*L0-L7*/	"","","","","","","","",
@@ -1125,7 +1107,7 @@ &gpio0 {
 	/*R0-R7*/	"","","","","","","","",
 	/*S0-S7*/	"","","","","","","","",
 	/*T0-T7*/	"","","","","","","","",
-	/*U0-U7*/	"","","","","","","","",
+	/*U0-U7*/	"","","","","P1_I3C_APML_ALERT_L","","","",
 	/*V0-V7*/	"","","","","","","","",
 	/*W0-W7*/	"","","","","","","","",
 	/*X0-X7*/	"","","","","","","","",
diff --git a/arch/arm/boot/dts/aspeed-bmc-amd-ruby.dts b/arch/arm/boot/dts/aspeed-bmc-amd-ruby.dts
index a7166512268f..655f408211cf 100644
--- a/arch/arm/boot/dts/aspeed-bmc-amd-ruby.dts
+++ b/arch/arm/boot/dts/aspeed-bmc-amd-ruby.dts
@@ -628,10 +628,6 @@ sbtsi_p0_0: sbtsi@4c,22400000000 {
         reg = <0x4c 0x224 0x00000000>;
         assigned-address = <0x4c>;
     };
-    sbrmi_p0_1: sbrmi@3c,22400000001 {
-        reg = <0x3c 0x224 0x00000001>;
-        assigned-address = <0x3c>;
-    };
 };

 #endif  // OPENBMC_I3C_ENABLE
@@ -644,11 +640,6 @@ sbtsi@4c {
 		compatible = "amd,sbtsi";
 		reg = <0x4c>;
 	};
-	// sbrmi
-	sbrmi@3c {
-		compatible = "amd,sbrmi";
-		reg = <0x3c>;
-	};
 };

 &espi_ctrl {
@@ -674,7 +665,7 @@ &gpio0 {
 	/*F0-F7*/	"","","","","","","","",
 	/*G0-G7*/	"","","","","","","","",
 	/*H0-H7*/	"","ASSERT_WARM_RST_BTN_L","ASSERT_SOC_RST_BTN_L","","","","","",
-	/*I0-I7*/	"","","","","","","","",
+	/*I0-I7*/	"","","","","","","","P0_I3C_APML_ALERT_L",
 	/*J0-J7*/	"","","","","","","","",
 	/*K0-K7*/	"","","","","","","","",
 	/*L0-L7*/	"","","","","","","","",
diff --git a/arch/arm/boot/dts/aspeed-bmc-amd-titanite.dts b/arch/arm/boot/dts/aspeed-bmc-amd-titanite.dts
index 501b01a441be..87ad302d7ebb 100644
--- a/arch/arm/boot/dts/aspeed-bmc-amd-titanite.dts
+++ b/arch/arm/boot/dts/aspeed-bmc-amd-titanite.dts
@@ -527,10 +527,6 @@ sbtsi_p0_0: sbtsi@4c,22400000000 {
         reg = <0x4c 0x224 0x00000000>;
         assigned-address = <0x4c>;
     };
-    sbrmi_p0_1: sbrmi@3c,22400000001 {
-        reg = <0x3c 0x224 0x00000001>;
-        assigned-address = <0x3c>;
-    };
 };

 &i3c5 {
@@ -547,10 +543,6 @@ sbtsi_p1_0: sbtsi@4c,22400000000 {
         reg = <0x4c 0x224 0x00000000>;
         assigned-address = <0x4c>;
     };
-    sbrmi_p1_1: sbrmi@3c,22400000001 {
-        reg = <0x3c 0x224 0x00000001>;
-        assigned-address = <0x3c>;
-    };
  };

 #endif  // OPENBMC_I3C_ENABLE
@@ -564,12 +556,6 @@ sbtsi@4c {
 		compatible = "amd,sbtsi";
 		reg = <0x4c>;
 	};
-
-	// sbrmi
-	sbrmi@3c {
-		compatible = "amd,sbrmi";
-		reg = <0x3c>;
-	};
 };

 &i2c3 {
@@ -581,12 +567,6 @@ sbtsi@48 {
 		compatible = "amd,sbtsi";
 		reg = <0x48>;
 	};
-
-	// sbrmi
-	sbrmi@38 {
-		compatible = "amd,sbrmi";
-		reg = <0x38>;
-	};
 };

 // Misc Regulators
@@ -838,7 +818,7 @@ &gpio0 {
 	/*F0-F7*/	"","","","","","","","",
 	/*G0-G7*/	"","","","","","","","",
 	/*H0-H7*/	"","ASSERT_WARM_RST_BTN_L","ASSERT_SOC_RST_BTN_L","","","","","",
-	/*I0-I7*/	"","","","","","","","",
+	/*I0-I7*/	"","","","","","","","P0_I3C_APML_ALERT_L",
 	/*J0-J7*/	"","","","","","","","",
 	/*K0-K7*/	"","","","","","","","",
 	/*L0-L7*/	"","","","","","","","",
@@ -850,7 +830,7 @@ &gpio0 {
 	/*R0-R7*/	"","","","","","","","",
 	/*S0-S7*/	"","","","","","","","",
 	/*T0-T7*/	"","","","","","","","",
-	/*U0-U7*/	"","","","","","","","",
+	/*U0-U7*/	"","","","","P1_I3C_APML_ALERT_L","","","",
 	/*V0-V7*/	"","","","","","","","",
 	/*W0-W7*/	"","","","","","","","",
 	/*X0-X7*/	"","","","","","","","",
--
2.25.1

